project/profiles/capacity4more/modules/c4m/user/c4m_user_profile/c4m_user_profile.module

```diff
@@ -383,6 +383,13 @@ function c4m_user_profile_form_user_register_form_alter(
     $form['account']['status']['#access'] = FALSE;
     // Alter 'notify user' checkbox title.
     $form['account']['notify']['#title'] = t('Send password reset link');
+
+    // When SA creates a user from 'admin/people/create' path, hide roles field,
+    // as SA may not create administrator. SA can edit user to assign him
```

Comment:
I read the comment on JIRA. I know there's a bug.
SA may not create admins straight away, to bypass this bug.